src/cloudai/systems/slurm/slurm_system.py

```
@@ -256,11 +270,12 @@ def is_job_running(self, job: BaseJob, retry_threshold: int = 3) -> bool:
             stdout, stderr = self.cmd_shell.execute(command).communicate()
             logging.debug(f"Job running: {command=} {stdout=} {stderr=}")
 
-            if "Socket timed out" in stderr or "slurm_load_jobs error" in stderr:
+            if self._is_transient_status_error(stderr):
                 retry_count += 1
                 logging.warning(
                     f"An error occurred while querying the job status. Retrying... ({retry_count}/{retry_threshold})."
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)